spec/suites/layer/TooltipSpec.js

```diff
@@ -461,6 +461,31 @@ describe('Tooltip', () => {
 		expect(tooltip.isOpen()).to.be.ok();
 	});
 
+	it("opens the tooltip if the tooltip is loaded while the map is dragging.", () => {
+		// simulate map dragging
+		map.dragging.moving = function () {
+			return true;
+		};
+
+		// If tooltips are dynamically loaded while the map is dragging, they need
+		// to be loaded when the dragging stops.
+		const layer = L.marker(center).bindTooltip('Tooltip', {permanent: true});
+		map.addLayer(layer);
+
+		// simulate map not dragging anymore
+		map.dragging.moving = function () {
+			return false;
+		};
```

Comment:
```suggestion
		map.dragging.moving = () => false;
```

Reply:
What happened to `L.Util.falseFn`?

Reply:
Yeah, I was thinking we can use that as well. Why don't we have a `L.Util.trueFn`?

Reply:
I made the suggested simplification to functions for this test, and in other `Tooltip` tests where this pattern occurs.

If you prefer that I use `L.Util.falseFn` and add an `L.Util.trueFn` and use that in these tests, just let me know and I will do that instead.